pkg/types/aws/platform.go

```diff
@@ -16,3 +19,9 @@ type Platform struct {
 	// +optional
 	DefaultMachinePlatform *MachinePool `json:"defaultMachinePlatform,omitempty"`
 }
+
+//SetBaseDomain parses the baseDomainID and sets the related fields on azure.Platform
```

Comment:
azure should be aws

Reply:
Not sure what happened here, out of date branch? The commit didn't touch this and in my branch it is aws?